nimble/host/include/host/ble_audio_common.h

```diff
@@ -47,6 +47,7 @@
                                                       __VA_ARGS__)
 
 #define BLE_BROADCAST_AUDIO_ANNOUNCEMENT_SVC_UUID                 0x1852
+#define BLE_BROADCAST_PUBLIC_BROADCAST_ANNOUNCEMENT_SVC_UUID      0x1856
```

Comment:
Shorter = better?
```suggestion
#define BLE_BROADCAST_PUB_BCST_ANNOUNCEMENT_SVC_UUID      0x1856
```
and probably doxygen comment with full name.